Spawn-App-iOS-SwiftUI/Views/SubComponents/TagButtonView.swift

```diff
@@ -25,7 +26,7 @@ struct TagButtonView: View {
                 .padding(.horizontal, 15)
                 .background{
                     Capsule()
-                        .fill(activeTag == mockTag ? Color(hex: "#173131") : .white)
+                        .fill(activeTag == mockTag ? Color(hex: "#E7E7DD") : .white)
```

Comment:
Not sure why you changed this, since it's what seems to have caused the selected tag color to differ from the figma design

Figma design:
![image](https://github.com/user-attachments/assets/3f8b594f-a54f-4bdc-bad7-dfcd339c1a14)

Reply:
Besides, we should use the `universalAccentColor` variable to modify this color, as it's used for the "hello, udhlee" title as well